packages/grid/x-data-grid-pro/src/index.ts

```diff
@@ -1,6 +1,13 @@
 import './typeOverloads';
 
-export { LicenseInfo } from '@mui/x-license';
+export {
+  /**
+   * @deprecated
+   * Use `@mui/x-license` package instead:
+   * import { LicenseInfo } from '@mui/x-license';
+   */
+  LicenseInfo,
+} from '@mui/x-license';
```

Comment:
This doesn't generate IntelliSense for me.

Actual:
<img src="https://github.com/mui/mui-x/assets/3165635/9f7d75c1-e034-41f4-996b-ccdc30d0bddf" width="567">

Expected:
<img src="https://github.com/mui/mui-x/assets/3165635/c1247944-8f74-4e73-8e5d-c9d7500a24fb" width="679">

How about?

```suggestion
import LicenseInfo from '@mui/x-license';

/**
  * @deprecated
  * Use `@mui/x-license` package instead:
  * import { LicenseInfo } from '@mui/x-license';
 */
export const LicenseInfo;
```

Reply:
Thanks, I couldn't figure it out!